src/Tests/Footer.test.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import {shallow} from 'enzyme';
+import Footer from '../Components/Footer'
+
+describe("Footer", () => {
+
+    const footer = shallow(<Footer />);
```

Comment:
I _think_ this might be valid syntax... But can we put this line in a `beforeEach`?

Reply:
Yes we can. Thanks for catching that.